app/controllers/CheckYourAnswersController.scala

```diff
@@ -36,10 +37,26 @@ class CheckYourAnswersController @Inject()(
   def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData) {
     implicit request =>
 
-      val list = SummaryListViewModel(
-        rows = Seq.empty
-      )
+      val operatePackagingSites: Option[(String, SummaryList)] = {
+        val list = OperatePackagingSitesSummary.summaryList(userAnswers = request.userAnswers, isCheckAnswers = true)
+        list.rows.headOption.fold(Option.empty[(String, SummaryList)])(_ => Some("operatePackagingSites.checkYourAnswersLabel" -> list))
+      }
+      val contractPacking: Option[(String, SummaryList)] = {
+        val list = ContractPackingSummary.summaryList(userAnswers = request.userAnswers, isCheckAnswers = true)
+        list.rows.headOption.fold(Option.empty[(String, SummaryList)])(_ => Some("contractPacking.checkYourAnswersLabel" -> list))
+      }
+      val imports: Option[(String, SummaryList)] = {
+        val list = ImportsSummary.summaryList(userAnswers = request.userAnswers, isCheckAnswers = true)
+        list.rows.headOption.fold(Option.empty[(String, SummaryList)])(_ => Some("imports.checkYourAnswersLabel" -> list))
+      }
+      val summaryList: Seq[(String, SummaryList)] = Seq(operatePackagingSites, contractPacking, imports).flatten
```

Comment:
I think this logic is better in a separate view file like we did in returns https://github.com/hmrc/soft-drinks-industry-levy-returns-frontend/blob/main/app/views/helpers/returnDetailsSummary.scala.html otherwise we are going to have to repeat all this for the registration confirmation page.

Reply:
Yeah this wouldvee been in the summary helpers but they were so complicated I didnt want to change initially. Ill refactor them and change their return types